.github/workflows/pipeline-deploy-bootstrap.yaml

```diff
@@ -18,3 +18,13 @@ jobs:
       with:
         ENV: test
       secrets: inherit
+    slack-notifications:
+      needs: [
+        deploy-bootstrap-dos-dev,
+      ]
```

Comment:
I think we should add deploy-bootstrap-dos-test to the needs list. Both the deploy dev and deploy test jobs will run in parallel and we want to know if either of them fails